File: lọc gauss/Python/gaussian filter.py
import cv2
import numpy
src = cv2.imread('lena.jpg', cv2.IMREAD_UNCHANGED)
dst = cv2.GaussianBlur(src,(5,5),cv2.BORDER_DEFAULT)
cv2.imshow("Gaussian Smoothing",numpy.hstack((src, dst)))
cv2.waitKey(0)
cv2.destroyAllWindows() 

File: lọc gauss/C++/gaussian filter/gaussian filter/gaussian filter.cpp
#include <opencv2/imgproc/imgproc.hpp>
#include <opencv2/opencv.hpp>
#include <opencv2/highgui/highgui_c.h>
using namespace cv;
int main(int argc, char** argv)
{
	namedWindow("original", CV_WINDOW_AUTOSIZE);
	Mat src = imread("lena.jpg", 1);
	Mat dst;
	imshow("original", src);
	for (int i = 1; i < 51; i = i + 2)
	{
		GaussianBlur(src, dst, Size(i, i), 0, 0);
		imshow("Gaussian filter", dst);
		waitKey(5000);
	}
}